data/content/agent_attributes.yaml

```
@@ -154,6 +154,12 @@ attributes:
   required: false
   desc: |
         The DogStatsD instance to send metrics to using UDP.
+- name: metrics-datadog-distributions
+  env_var: BUILDKITE_METRICS_DATADOG_DISTRIBUTIONS
+  default_value: "false"
+  requires: false
```

The linting picked up a typo here with the field name:

```suggestion
  required: false
```